src/common/dao/webhook_job.go

```diff
@@ -0,0 +1,127 @@
+package dao
+
+import (
+	"fmt"
+
+	"github.com/astaxie/beego/orm"
+	"github.com/goharbor/harbor/src/common/models"
+	"github.com/goharbor/harbor/src/common/utils/log"
+)
+
+// UpdateWebhookJob update webhook job
+func UpdateWebhookJob(job *models.WebhookJob, props ...string) (int64, error) {
```

Comment:
Can we make this func internal, or do not expose the column names out of the package?

Reply:
Per discussion, this will not be fixed in this PR.